pkg/operator/ceph/cluster/watcher.go

```diff
@@ -130,6 +148,277 @@ func (c *clientCluster) onK8sNode(ctx context.Context, object runtime.Object) bo
 	return false
 }
 
+func (c *clientCluster) handleNodeFailure(ctx context.Context, cluster *cephv1.CephCluster, node *v1.Node) error {
+	watchForNodeLoss, err := k8sutil.GetOperatorSetting(ctx, c.context.Clientset, opcontroller.OperatorSettingConfigMapName, "ROOK_WATCH_FOR_NODE_FAILURE", "true")
```

Comment:
We need this env var to be in the settings configmap (in operator.yaml and operator-openshift.yaml), with the default value `false`. Then it's easy for the user to find it if needed to enable the feature.

Reply:
correction, I believe we discussed the default is `true` as you also have it here.

Reply:
added